pkg/services/annotations/annotationsimpl/composite_store_test.go

```diff
@@ -0,0 +1,68 @@
+package annotationsimpl
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/grafana/grafana/pkg/services/annotations"
+	mock "github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCompositeStore_Get(t *testing.T) {
+	t.Run("should join errors", func(t *testing.T) {
```

Comment:
Can we add a couple extra tests to cover the dispatching?

- One that calls `Get()` and asserts that both stores got called.
- One that calls some write method and asserts that only the primary store got called.

Reply:
> One that calls Get() and asserts that both stores got called.

We can, but I'm not sure what that gives us that asserting the final output contains the results of both stores.